doc/guides/connection_management.rst

```diff
@@ -1,4 +1,4 @@
-.. _connection_mgnt:
+.. _connection_management:
```

Comment:
Is this required at all? I thought Sphinx automatically creates anchors for titles?

Reply:
I think it does. However, I'd rather rely on a static name for references, instead of one that changes when the title changes...